doc/api/documentation.markdown

```diff
@@ -42,34 +42,20 @@ compatibility should not be expected.
 
 ```
 Stability: 1 - Experimental
-This feature was introduced recently, and may change
-or be removed in future versions.  Please try it out and provide feedback.
-If it addresses a use-case that is important to you, tell the node core team.
+This feature is subject to change, and is gated by a command line flag.
+It may change or be removed in future versions.
 ```
 
 ```
-Stability: 2 - Unstable
-The API is in the process of settling, but has not yet had
-sufficient real-world testing to be considered stable. Backwards-compatibility
-will be maintained if reasonable.
+Stability: 2 - Stable
+The API has proven satisfactory. Compatibility with the npm ecosystem
+is a high priority, and will not be broken unless absolutely necessary.
 ```
 
 ```
-Stability: 3 - Stable
-The API has proven satisfactory, but cleanup in the underlying
-code may cause minor changes.  Backwards-compatibility is guaranteed.
```

Comment:
How do I, as a module author, assert this semver in my `package.json`? There is no `engines.iojs` field I'm' aware of.